apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -15,9 +16,20 @@ import ListBoxFooter from './components/ListBoxFooter';
 import getScrollIndex from './interactions/listbox-get-scroll-index';
 import getFrequencyAllowed from './components/grid-list-components/frequency-allowed';
 import useFrequencyMax from './hooks/useFrequencyMax';
+import { ScreenReaderForSelections } from './components/ScreenReaders';
+import InstanceContext from '../../contexts/InstanceContext';
 
 const DEFAULT_MIN_BATCH_SIZE = 100;
 
+const StyledWrapper = styled(Box)(() => ({
+  [`& .screenReaderOnly`]: {
+    position: 'absolute',
+    height: 0,
+    width: 0,
+    overflow: 'hidden',
+  },
+}));
```

Comment:
maybe use
```
const ScreenReaderOnlyText = styled('div')(() => {
    position: 'absolute',
    height: 0,
    width: 0,
    overflow: 'hidden',
});
```
instead (I thing it would be more generic and reusable)